src/integrations/prefect-redis/prefect_redis/messaging.py

```diff
@@ -380,6 +389,15 @@ async def _send_to_dlq(self, msg: RedisStreamsMessage, retry_count: int):
         # Add to a Redis set for easy retrieval
         await redis_client.sadd(self.subscription.dlq_key, message_id)
 
+    async def trim_stream_if_necessary(self) -> None:
```

Comment:
Looks like this method is primarily used internally for the `Consumer` class, so it'd be nice to mark it as private-ish.
```suggestion
    async def _trim_stream_if_necessary(self) -> None:
```